src/main/resources/sql/Create_DB.sql

```diff
@@ -25,6 +32,10 @@ CREATE TABLE Users (
     Pass_id int
         REFERENCES Passport(Id)
         ON DELETE NO ACTION
+        ON UPDATE CASCADE,
+    Driving_license_id int
+        REFERENCES Driving_license(Id)
+        ON DELETE NO ACTION
         ON UPDATE NO ACTION,
```

Comment:
Думаю стоит сделать как в Passport(Id), чтобы избежать потерю или перепутывания прав между пользователями:
```suggestion
        ON UPDATE CASCADE,
```